src/listener/server-profiles.js

```diff
@@ -132,10 +132,15 @@ export default class ServerProfilesListener extends Listener {
     let profileHyperlink = deviceAndHyperlink.hyperlink;
     this.transform.text(msg, "Ok " + msg.message.user.name + " I am going to power on the server profile " + this.transform.hyperlink(profileHyperlink, profileName) + ".  Are you sure you want to do this? (@" + this.robot.name + " yes/@" + this.robot.name + " no)");
 
-
     dialog.addChoice(/yes/i, () => {
       this.client.ServerProfiles.getServerProfile(msg.profileId).then((res) => {
-        return this.serverHardware.PowerOnHardware(res.serverHardwareUri, msg);
+        if (res.serverHardwareUri === null) {
+          return this.transform.text(msg, msg.message.user.name + ", " + this.transform.hyperlink(profileHyperlink, profileName) + " does not have any assigned server hardware to power on. Try assigning server hardware to the profile or try powering on a different profile.");
```

Comment:
I would remove this from the last sentence "or try powering on a different profile."